01_temperature_processing.R

```diff
@@ -44,7 +45,7 @@ library(plyr)
 
 #Set the data file path and file that you want to process
 src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'
-SubID <- 'XXXX' # Enter the submission ID from VolWQDB
+SubID <- '0020' # Enter the submission ID from VolWQDB
```

Comment:
I think it would be a good improvement.  To would just require some additions to the SiteMasterInfo file.  It could provide input to R that I need further in the data flow (03b) and provide more accurate and complete data to the t_Submissions table.  It would require appending the info to the table, then retrieving the new submission ID number.

Steve Hanson
ODEQ Laboratory
Ph: 503.693.5737

From: petertbryant [mailto:notifications@github.com]
Sent: Friday, August 05, 2016 9:29 AM
To: petertbryant/ContinuousDataReview
Cc: HANSON Steve; Mention
Subject: Re: [petertbryant/ContinuousDataReview] Comments for grade changes (#3)


In 01_temperature_processing.R<https://github.com/petertbryant/ContinuousDataReview/pull/3#discussion_r73720859>:

> @@ -44,7 +45,7 @@ library(plyr)

>

>  #Set the data file path and file that you want to process

>  src_file <- '//deqlead02/Vol_Data/salmon-drift/2015/Fake2SDCWC2015CDO.xls'

> -SubID <- 'XXXX' # Enter the submission ID from VolWQDB

> +SubID <- '0020' # Enter the submission ID from VolWQDB

Ok. I think I need to see the workflow written out to get a complete grasp on this. I think it's probably fine the way it is here for now.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/petertbryant/ContinuousDataReview/pull/3/files/5e0a4e5e7e6880fd411f9290baa0ef06d04cfeb8#r73720859>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATJGv5trXm9IZ28bqCXfjHhYCv3AsvOCks5qc2SwgaJpZM4Jb301>.